test/cloudtest/pkg/utils/utils.go

```diff
@@ -9,3 +15,14 @@ func Contains(a []string, x string) bool {
 	}
 	return false
 }
+
+// NewRandomStr - generates random string of desired length, size should be multiple of two for best result.
```

Comment:
Could this be useful to the borader project? Shall we move it to the root utils?

Reply:
I suppose finally yes, but for separate project it should be at least documented and finally